### PR TITLE
docs: update controller to use adminAPIVersion=v3

### PR DESCRIPTION
**Pull request by anhldbk**
Hi,

This PR is made according to [our discussion](https://github.com/apache/apisix-ingress-controller/issues/1435#issuecomment-1374964779)

Thank you.

### CODE REVIEW

**Comment by anhldbk** [comment]
@tao12345666333 PTAL

File: docs/en/latest/deployments/ack.md [hunk 1]
```diff
@@ -44,14 +44,15 @@ The script below installs APISIX and the ingress controller:
...
   --set gateway.type=LoadBalancer \
   --set ingress-controller.enabled=true \
   --set etcd.persistence.storageClass="alicloud-disk-ssd" \
   --set etcd.persistence.size="20Gi" \
+  --create-namespace \
   --namespace ingress-apisix \
-  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
+  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix \
+  --set ingress-controller.config.apisix.adminAPIVersion=v3
 kubectl get service --namespace ingress-apisix
```
Comment (tokers) [inline docs/en/latest/deployments/ack.md:56]: We may add a note here to say we use APISIX 3.0 as the example so we set the `adminAPIVersion` to `v3`. If you're using APISIX 2.x, please reset it to `v2`.

Comment (anhldbk) [reply docs/en/latest/deployments/ack.md:56]: @tokers Should it look like this?

```bash
#  We use Apisix 3.0 as the example. If you're using Apisix v2.x, please set to v2
ADMIN_API_VERSION=v3
helm install apisix apisix/apisix \
  --set gateway.type=LoadBalancer \
  --set ingress-controller.enabled=true \
  --set etcd.persistence.storageClass="alicloud-disk-ssd" \
  --set etcd.persistence.size="20Gi" \
  --create-namespace \
  --namespace ingress-apisix \
  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix \
  --set ingress-controller.config.apisix.adminAPIVersion=$ADMIN_API_VERSION
```

Comment (tao12345666333) [reply docs/en/latest/deployments/ack.md:56]: @anhldbk good idea! I think that's fine.  
cc @navendu-pottekkat

Comment (anhldbk) [reply docs/en/latest/deployments/ack.md:58]: @tao12345666333 @tokers @navendu-pottekkat Updated. PTAL

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Review by navendu-pottekkat — APPROVED** [review]

**Review by shreemaan-abhishek — APPROVED** [review]

**Comment by anhldbk** [comment]
@tao12345666333 why is this change not released? Still, I see the latest version on our docs site is v1.6.1

**Comment by tao12345666333** [comment]
@anhldbk The v1.6.1 is still in the vote process. https://lists.apache.org/thread/wc9o6b4sndkpkryw66jgkc1kmrrs2vh2

**Pull request merged**